task/reindex.go

```diff
@@ -649,9 +659,49 @@ func convertToSearchDataModel(searchDataImport extractorModels.SearchDataImport)
 			RawLabel: dim.RawLabel,
 		})
 	}
+
+	if enableTopicTagging {
```

Comment:
This would probably be better done in the transformZebedeeDoc function. Or called from it anyway. 

By abstracting it out we *could* add this to dataset-api items later but at the moment it's not adding value I don't think. 

This function is about converting to the search data model so by putting additional processing here you're conflating what it's doing.